.travis.yml

```diff
@@ -4,7 +4,7 @@ services:
   - redis-server
 language: go
 go:
-  - 1.13.1
+  - 1.13.x
```

Comment:
I would not like to add this change as I want to upgrade go versions manually and keep CI builds reproducible. And this version can change separately from Dockerfiles.